public/index.html

```diff
@@ -22,23 +24,25 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-  <title>VEGE App</title>
-</head>
+    <title>VEGE App</title>
+  </head>
 
-<body id="body" style="
-height:100%;
-background-image: url('https://images.unsplash.com/photo-1498579809087-ef1e558fd1da?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1950&q=80');
-  background-size: cover;
-  background-position: center center;
-  background-repeat:  no-repeat;
-  background-attachment: fixed;
- background-color:   lightgray
-;
-">
-<!-- #FDB3BC #F27881 #FFF1F5 #FBDFE7 #EF5B67 #FBBBC1 -->
-  <noscript>You need to enable JavaScript to run this app.</noscript>
-  <div id="root" style="height: 100vh"></div>
-  <!--
+  <body
+    id="body"
+    style="
```

Comment:
Huh, why is the styling even here inlined? I guess you could make a card to move this to the stylesheet. Also have image in the assets folder downloaded, as you never know when unsplash link can change :)

Reply:
good points, added these to the backlog!